### PR TITLE
Add kn-plugin-func formula

### PR DESCRIPTION
**Pull request by dsimansk**
# Changes

- :gift: Add kn-plugin-func formula (latest)

/cc @rhuss @maximilien 

FYI @omerbensaadon @lance

### CODE REVIEW

**Review by rhuss — COMMENTED** [review]
/approve
/lgtm

File: func.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  homepage "https://github.com/knative-sandbox/#{path_name}"
+
+  version v
+
+  if OS.mac?
+    url "#{base_url}/#{file_name}_darwin_amd64.gz"
+    sha256 "f83d1ff15045cd882fedda4944581d5e3084a82d68a309b4c5c900af91968e11"
+  else
+    url "#{base_url}/#{file_name}_linux_amd64.gz"
```
Comment (lance) [inline func.rb:18]: Just curious - does this also work for Windows Subsystem for Linux?

Comment (rhuss) [reply func.rb:18]: tbh, no idea whether brew is available for Windows Subsystem. Afaik Windows leverages a different package manager, https://chocolatey.org/. We might consider supporting this, too (but I never worked with it)

Comment (dsimansk) [reply func.rb:18]: IMO it should work with Windows Subsystem for Linux, once you'd have Homebrew installed. A while ago I've used native Windows manages that was in beta back then, so.... We might want to open that Pandora box. :)

Comment (lance) [reply func.rb:18]: Seems like it should just work? https://medium.com/@edwardbaeg9/using-homebrew-on-windows-10-with-windows-subsystem-for-linux-wsl-c7f1792f88b3

**Pull request merged**